ghost/core/core/server/data/migrations/versions/5.60/2023-08-16-08-45-37-add-collections-meta-fields.js

```diff
@@ -0,0 +1,75 @@
+const {combineNonTransactionalMigrations, createAddColumnMigration} = require('../../utils');
+
+module.exports = combineNonTransactionalMigrations(
+    createAddColumnMigration('collections', 'og_image', {
+        type: 'string',
+        maxlength: 2000,
+        nullable: true
+    }),
+    createAddColumnMigration('collections', 'og_title', {
+        type: 'string',
+        maxlength: 300,
+        nullable: true
+    }),
+    createAddColumnMigration('collections', 'og_description', {
+        type: 'string',
+        maxlength: 500,
+        nullable: true
+    }),
+    createAddColumnMigration('collections', 'twitter_image', {
+        type: 'string',
+        maxlength: 2000,
+        nullable: true
+    }),
+    createAddColumnMigration('collections', 'twitter_title', {
+        type: 'string',
+        maxlength: 300,
+        nullable: true
+    }),
+    createAddColumnMigration('collections', 'twitter_description', {
+        type: 'string',
+        maxlength: 500,
+        nullable: true
+    }),
+    createAddColumnMigration('collections', 'meta_title', {
+        type: 'string',
+        maxlength: 2000,
+        nullable: true,
+        validations: {
+            isLength: {
+                max: 300
+            }
+        }
+    }),
+    createAddColumnMigration('collections', 'meta_description', {
+        type: 'string',
+        maxlength: 2000,
+        nullable: true,
+        validations: {
```

Comment:
hm. true. we do use this syntax in other migrations too. I think it's here to match the syntax in `schema` to dot?

Reply:
Yeah I've seen it in other migrations, AFAIK they aren't there for a reason though, just misunderstanding or copy/paste errors!

IMO it will cause confusion by being there but :shrug: